service/aas/authorization/pom.xml

```diff
@@ -25,8 +25,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
Maybe we can remove compiler plugin definition from here? We already have it in parent pom.
Beside that there is typo in goupID, should be:
<groupId>org.apache.maven.plugins</groupId>
and there is
<groupId>org.apache.maven.plugin</groupId>

Reply:
made